### DISCUSSION TITLE
Feature - Allow to change existing field to M2O

### DISCUSSION THREAD

**Discussion by dimitrov-adrian** [directus/directus#3031] [Feature Requests]
If import tables with relational data, there is no way to "link" the relations since the key fields are set to Integer or whatever they are and no option to turn them into relations.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
